src/api/endpoints/redemption_requests.py

```diff
@@ -38,16 +56,18 @@ def post(cls):
                 name=name,
                 value=value,
                 user=g.current_user,
-                country=country
+                country=country,
+                society=g.current_user.society
```

Comment:
Is it necessary to add this when we can query the Society from the user?
Why add more relationships?